### ISSUE TITLE
example app

### ISSUE THREAD

**Issue by dionassydqi** [yale8848/CacheWebView#146]
this is a good library, can you make a sample project? I want to apply it in my application, I am making a browser application based on webview

**Comment by yale8848** [comment]
you can clone this project, than you can find app is the sample project.

**Comment by dionassydqi** [comment]
Yes I have tried using the library, everything went smoothly, but I can't see a significant difference between using this library or not. I have followed the instructions by entering this code - >> public boolean should OverrideUrlLoading (WebView view, WebResourceRequest request), and so according to the instructions and this code on the main activity
     WebViewCacheInterceptorInst.getInstance (). init (new WebViewCacheInterceptor.Builder (this));

WebViewCacheInterceptor.Builder builder = new WebViewCacheInterceptor.Builder (this);

    builder.setCachePath (new File (this.getCacheDir (), "cache_path_name")) // set cache path, default getCacheDir, name CacheWebViewCache
                       .setCacheSize (1024 * 1024 * 100) // set cache size, default is 100M
                       .setConnectTimeoutSecond (20) // set http connect timeou, default 20 seconds
                       .setReadTimeoutSecond (20) // set http read timeout, default 20 seconds
                       .setCacheType (CacheType.NORMAL); // set capital cache is normal, default is force cache static capital

    WebViewCacheInterceptorInst.getInstance (). Init (builder);

Is that enough to work well with this library?

**Comment by dionassydqi** [comment]
sorry if I look stupid, because I just learned android studio,

**Comment by dionassydqi** [comment]
` WebViewCacheInterceptorInst.getInstance().init(new WebViewCacheInterceptor.Builder(this));
        WebViewCacheInterceptor.Builder builder =  new WebViewCacheInterceptor.Builder(this);

        builder.setCachePath(new File(this.getCacheDir(),"cache_path_name"))//set cache path, default getCacheDir, name CacheWebViewCache
                .setCacheSize(1024*1024*100)//set cache size, default 100M
                .setConnectTimeoutSecond(20)//set http connect timeou,default 20 seconds
                .setReadTimeoutSecond(20)//set http read timeout,default 20 seconds
                .setCacheType( CacheType.NORMAL);//set cache modal is normal, default is force cache static modal

        WebViewCacheInterceptorInst.getInstance().init(builder);`

**Comment by yale8848** [comment]
Yes , of course it is nothing different with system webview when you glance this lib. There are 3 reasons for use this lib:

1. let WebView cache space more bigger; // system webview can't set space size
2. force cache static, it will more fast; // if server not set cache header properly, system webview have more http request
3. want to get web page resource in cache, e.g , get pic in cache;//system webview can't

actually, if you not foucs these feature，you may not use this lib, just WebView is ok.

**Comment by AaronHuange** [comment]
I do it as same as dionassydqi,I enter the web page again when i offline.But WebView cannot Off-line load It
